package.json

```diff
@@ -78,19 +78,19 @@
     "start-server-and-test": "^1.14.0"
   },
   "scripts": {
-    "dev": ". ./scripts/populate-git-env.sh && next dev -p 3100",
-    "debug": ". ./scripts/populate-git-env.sh && WDYR=1 NODE_OPTIONS='--inspect' next dev -p 3100",
+    "dev": "next dev -p 3100",
+    "debug": "WDYR=1 NODE_OPTIONS='--inspect' next dev -p 3100",
     "start": "next start -p 3100",
-    "build": ". ./scripts/populate-git-env.sh && next build",
-    "export": ". ./scripts/populate-git-env.sh && next export",
+    "build": "next build",
+    "export": "next export",
     "lint": "next lint",
     "test:e2e": "yarn build && start-server-and-test start http://localhost:3100 cypress:run",
     "cypress:run": "cypress run",
     "test": "yarn run test:e2e",
     "script:build-translations": "node ./scripts/build-translations.js",
     "script:extract-i18n-keys": "yarn build && node ./scripts/extract-react-intl-keys.js",
-    "script:populate-git-env:print": ". ./scripts/populate-git-env.sh && echo 'SHA: '${GIT_COMMIT_SHA} && echo 'SHA (short): '${GIT_COMMIT_SHA_SHORT} && echo 'REF (current branch/tag): '${GIT_COMMIT_REF} && echo 'TAGS: '${GIT_COMMIT_TAGS}",
```

Comment:
Before merging you should delete the script if it's no longer being used.

Reply:
Removed

Reply:
We can also remove `/scripts/populate-git-env.sh` if it's not being used anymore.

Reply:
checked - not used anywhere.
So, removed it